js/src/widgets/metadataView.js

```diff
@@ -164,11 +163,7 @@
     var label = attr;
 
     label = label.replace(/^@/, '');
-    label = label.replace(/([A-Z])/g, ' $1');
     label = label.replace(/\s{2,}/g, ' ');
-    label = label.replace(/\w\S*/g, function(txt) {
```

Comment:
Any update on this? The CSS should obviously removed, but in my opinion title-casing should be done by the entity creating the manifests and not by the client.